### ISSUE TITLE
WCARO Togo Data mismatch after syncing client app and records on server

### ISSUE THREAD

**Issue by ageryck** [opensrp/opensrp-client-chw#1857]
A supervisor level user (CHSS) provider_id `mtchara82` synced to server and had the following numbers after sync;
![image](https://user-images.githubusercontent.com/23252704/129020670-4db1ecdc-447c-4822-b5da-e9a17cc10bdf.png)
Upon syncing separate devices with the above user credentials the results are consistent on various devices as shared below;
![image](https://user-images.githubusercontent.com/23252704/129020835-04cce2b8-1840-4742-a455-91113ede6d00.png)

A further test with various [chw](https://docs.google.com/spreadsheets/d/1b-uZinHcROHpJeSffw0EbgNEQZ9ygnb9_rHTuWVMWLc/edit#gid=0) under the same team (from which the above supervisor comes from) we only had on match and other showing more records on device compared to server results.

The issue is to investigate the reason/justification for the difference.

**Comment by ageryck** [comment]
While investigating the above issue we equally found that for various [CHW's](https://docs.google.com/spreadsheets/d/1b-uZinHcROHpJeSffw0EbgNEQZ9ygnb9_rHTuWVMWLc/edit#gid=0) under this region belonging to this team have number above the server figures upon sync

**Comment by ageryck** [comment]
For CHW's after conducting several tests and tracking down team assignments we found the following;
1. On team assignment, several teams had been assigned to more than one location (2 locations), on the first attempt this did not give results when querying server. Our Initial approach was to query which team the provider belongs to and track down the location the provider belongs this resulted into two teams as was seen on web too, upon editing this to the correct team the problem persisted. We then hit the locations endpoint kin to what the client device would do and we got results of two locations still. This necessitated further investigation which led to the realization that there was another complete different location that shared team with this provider and this was the case for a number of providers. Upon editing this independent location team assignment, we were able to get correct results. In discussion with @dubdabasoduba we made the following two recommendations for OpenSRP web improvement to support such future investigations;
- [ ] Change the first column on the team assign to clearly indicate as `Location Name` currently it indicates just `Name`, this is confusing as locations and teams share similar names
- [ ] Create two views, Filter by teams to know which locations have this team been assigned and filter by user to list associated teams
